### PR TITLE
new module gatk4/svcluster

### LINKED ISSUE #2308: new module: gatk4/svcluster
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #2308  

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by matthdsm — APPROVED** [review]

File: modules/nf-core/gatk4/svcluster/meta.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: "gatk4_svcluster"
+description: Clusters structural variants based on coordinates, event type, and supporting algorithms
+keywords:
+  - sort
```
Comment (ramprasadn) [inline modules/nf-core/gatk4/svcluster/meta.yml:4]: I think this needs to be changed?

Comment (nvnieuwk) [reply modules/nf-core/gatk4/svcluster/meta.yml:4]: Yeah you're right! Nice catch!

Comment (nvnieuwk) [inline modules/nf-core/gatk4/svcluster/meta.yml:4]:
```suggestion
  - gatk4
  - svcluster
  - vcf
  - structural variants
```

**Review by ramprasadn — APPROVED** [review]
LGTM! 🚀

**Pull request merged**